mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -235,6 +236,23 @@ static LogicalResult checkLDSSize(OpT op, int64_t aBufferBytes,
   return success(ldsBytes <= 64 * 1024);
 }
 
+Value gpuAlloc(OpBuilder &b, Location loc, int bufferDim, Type elementType,
```

Comment:
Perhaps `int64_t` or `size_t` for `bufferDim`

Also, instead of a weird bool, maybe an enum?

(Heck, maybe importing the address space enum with some `using`s so you can just say `Workgroup` and `Private`)